wls-broadcast-service/src/test/java/de/muenchen/oss/wahllokalsystem/broadcastservice/integration/BroadcastIntegrationTest.java

```diff
@@ -0,0 +1,296 @@
+import de.muenchen.oss.wahllokalsystem.broadcastservice.service.BroadcastMapper;
+import de.muenchen.oss.wahllokalsystem.broadcastservice.utils.BroadcastSecurityUtils;
+import de.muenchen.oss.wahllokalsystem.broadcastservice.utils.Testdaten;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.FachlicheWlsException;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.TechnischeWlsException;
+import jakarta.servlet.ServletException;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers;
+import org.springframework.security.web.FilterChainProxy;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.servlet.resource.NoResourceFoundException;
+
+@Slf4j
+@SpringBootTest(
+        classes = { MicroServiceApplication.class },
+        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+        properties = {
+                "spring.datasource.url=jdbc:h2:mem:wahllokalsystem;DB_CLOSE_ON_EXIT=FALSE",
+                "refarch.gracefulshutdown.pre-wait-seconds=0"
+        }
+)
+@ActiveProfiles(profiles = { SPRING_TEST_PROFILE, SPRING_NO_SECURITY_PROFILE })
+public class BroadcastIntegrationTest {
+
+    @Autowired
+    WebApplicationContext context;
+
+    @Autowired
+    FilterChainProxy springSecurityFilterChain;
+
+    @Autowired
+    MessageRepository messageRepository;
+
+    @Autowired
+    BroadcastMapper bcMapper;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    private AutoCloseable closeable;
+
+    MockMvc mvc;
+
+    @Value("${local.server.port}")
+    private int port;
+
+    @Value("${service.info.oid}")
+    private String serviceOid;
+
+    private static final String BROADCAST_URL = "/businessActions/broadcast";
+    private static final String GETMESSAGE_URL = "/businessActions/getMessage/";
+    private static final String DELETE_URL = "/businessActions/messageRead/";
+
+    private static final List<String> WAHLBEZIRK_I_DS = Arrays.asList("1", "2", "3");
+    private static final BroadcastMessageDTO BROADCAST_MESSAGE_DTO = new BroadcastMessageDTO(WAHLBEZIRK_I_DS, "Das ist ein Test");
+
+    @BeforeEach
+    public void setup() {
```

Comment:
Schreibt man mit Junit 5 nicht mehr

```suggestion
    void setup() {
```